pkg/pmem-csi-driver/controllerserver-node.go

```diff
@@ -267,6 +267,7 @@ func (cs *nodeControllerServer) DeleteVolume(ctx context.Context, req *csi.Delet
 		}
 	} else {
 		glog.V(3).Infof("Volume %s not found in driver cache.", req.VolumeId)
+		return &csi.DeleteVolumeResponse{}, nil
```

Comment:
Same here, you can remove the `glog.V(3).Infof`. It's just going to add noise to the log when calls are repeated.

Reply:
Done